src/coreclr/jit/emitxarch.cpp

```diff
@@ -11104,8 +11130,9 @@ BYTE* emitter::emitOutputSV(BYTE* dst, instrDesc* id, code_t code, CnsVal* addc)
 
         // Use the large version if this is not a byte
         // TODO-XArch-Cleanup Can the need for the 'w' size bit be encoded in the instruction flags?
-        if ((size != EA_1BYTE) && (ins != INS_imul) && (ins != INS_bsf) && (ins != INS_bsr) && (!insIsCMOV(ins)) &&
-            !IsSSEInstruction(ins) && !IsAVXInstruction(ins))
+        // Anthony: I believe we may remove !insIsCMOV check, but leaving for comparison purposes with another
+        // similar check below
```

Comment:
Nits: Seems like the TODO above is outdated with this change, remove it?
Also, I would prefer to not leave author names in comments and to write them in third person or plural first person instead.

Reply:
Thanks, I have removed the todo and my comment.